boards/esp32-ethernet-kit/doc.txt

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
```

Comment:
Please add your copyright notice in all files that you have copied and modified.